test/integration/scheduler_perf/BUILD

```diff
@@ -20,7 +20,10 @@ go_library(
         "//staging/src/k8s.io/client-go/informers/core/v1:go_default_library",
         "//staging/src/k8s.io/client-go/kubernetes:go_default_library",
         "//staging/src/k8s.io/client-go/rest:go_default_library",
+        "//staging/src/k8s.io/component-base/metrics/legacyregistry:go_default_library",
         "//test/integration/util:go_default_library",
+        "//vendor/github.com/prometheus/client_model/go:go_default_library",
```

Comment:
This dependency is breaking the build. See https://github.com/kubernetes/kubernetes/pull/84302.

Unclear why this wasn't caught by the presubmit... cc @BenTheElder

Reply:
I opened https://github.com/kubernetes/test-infra/pull/16168 to add coverage. Apparently we miss this in the build presubmit.

Reply:
can we revert this to unbreak master, then reintroduce when fixed?

Reply:
@serathius so what's the right way to import `github.com/prometheus/client_model/go` ? I am going through #84302 and what's link in there though it may take some time to figure out what to do.

Reply:
You should not use or depend on internals of prometheus. 
Please move functions that rely on prometheus internals into staging/src/k8s.io/component-base/metrics/testutil
Example PR https://github.com/kubernetes/kubernetes/pull/85289

Reply:
@serathius @liggitt @mikedanese Moving the code under `staging/src/k8s.io/component-base/metrics/testutil` in https://github.com/kubernetes/kubernetes/pull/87923